ansible/roles/dev/tasks/main.yml

```diff
@@ -175,3 +175,23 @@
 - name: Install pulp-dev library (system-wide)
   command: pip3 install -e {{pulp_devel_dir}}/devel/pulp-dev
   when: "{{ ['devel'] | issubset(pulp_available_repositories) }}"
+
+- name: install systemd files
+  copy:
```

Comment:
Also, since this is a developer install, the files (or softlinks) should be installed in ```/etc/systemd/system``` where user-overridden config files go.  This would prevent any config files left by installing the actual package from overwriting our own.